.circleci/config.yml

```diff
@@ -343,7 +289,29 @@ jobs:
             echo "password: $CONBENCH_PASSWORD" >> .conbench
             echo "host_name: CircleCI-runner" >> .conbench
 
-            conbench cpp-micro --run-name="commit: $(git rev-parse HEAD)" --run-reason="commit"
+            # figure out run name/reason
+            if [[ $CIRCLE_BRANCH == 'main' ]]; then
+              run_name="commit: $(git rev-parse HEAD)"
+              run_reason='commit'
+            else
+              run_name="pull request $CIRCLE_PR_NUMBER: $(git rev-parse HEAD)"
```

Comment:
Several of us force push and amend onto a single commit; Will that show up as 1 run or multiple runs ?

Reply:
The intention is that benchmarking will be run every time a commit is pushed, and not more or less. So if you have a PR open and do something like this to your branch:
```
git commit --amend
git push --force
```
that should start one new build, which means one new run. The old run won't be deleted. Does that answer your question?

Reply:
Thats fine; my concern was whether different commits due to an amend will show up as one.